template/page-docs.html.jinja

```diff
@@ -92,7 +92,7 @@
       },
       {
         "description": _("XRP Faucet"),
-        "link": "./xrp-testnet-faucet.html",
+        "link": ".xrp-testnet-faucet.html",
```

Comment:
Revert this change too. (`.xrp-testnet-faucet.html` with the leading `.` and no slash is not right either)